docs/typescript.md

```diff
@@ -0,0 +1,149 @@
+---
+permalink: /typescript
+title: TypeScript
+---
+
+# Typescript
+
+CodeceptJS supports [type declaration](https://github.com/codecept-js/CodeceptJS/tree/master/typings) for [TypeScript](https://www.typescriptlang.org/). It means that allow you to write your tests in TypeScript. Also, all of your custom steps could be write in TypeScript.
```

Comment:
It means now you can write your tests in TS. Also, all of your custom steps could be written in TS